framework/src/executioners/Transient.C

```diff
@@ -440,6 +447,24 @@ Transient::solveStep(Real input_dt)
 
       _solution_change_norm = _problem.solutionChangeNorm();
 
+      // Check for stop condition based upon steady-state check flag:
+      if (!_xfem_repeat_step &&
+          _trans_ss_check == true && _time > _ss_tmin)
+      {
+        // Compute new time solution l2_norm
+        NonlinearSystem & nl = _problem.getNonlinearSystem();
+        NumericVector<Number> & current_solution  = (*nl.sys().current_local_solution);
+        NumericVector<Number> & old_solution = (*nl.sys().old_local_solution);
+        NumericVector<Number> & difference = *NumericVector<Number>::build(_communicator).release();
```

Comment:
Yeah I know, I was just testing you. I borrowed it from your code in `FEProblem::solutionChangeNorm()` ;-D